### PR TITLE
Route registrar job in service-fabrik-boshrelease

### PR DESCRIPTION
**Pull request by sauravmndl**
route_registrar job from routing-release doesn't support to take host as input. Added a new job in sf bosh release to support that.

### CODE REVIEW

File: jobs/route_registrar/monit [hunk 1]
```diff
@@ -0,0 +1,5 @@
+check process route_registrar
```
Comment (kamath-prasad) [inline jobs/route_registrar/monit]: just a question , will there be a dependency between this process and broker process ?

Comment (sauravmndl) [reply jobs/route_registrar/monit]: No. there won't be any dependency monit process wise. But in health check of route_registrar, it looks for a file placed :https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/blob/master/jobs/service-fabrik-broker/templates/bin/health_check.erb
Also health check looks for 9292 and 9293 port to be up.  So if broker is up and running then only route registration happens successfully. But process runs in wither cases. You see any concern?

Comment (kamath-prasad) [reply jobs/route_registrar/monit]: dont see any concern's since our last conversation.

**Comment by ketakig12** [comment]
@sauravmndl Do we then need to change [release for route_registrar](https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/blob/eb4609ac3bd46a88d386d059e0be18077aa4aa2e/templates/deployment.yml#L230) to **service-fabrik** ?

**Comment by sauravmndl** [comment]
@ketakig12 Users consuming the bosh release can still use same sample manifest to deploy service-fabrik. For HA solution user can use the new job introduced in service-fabrik-boshrelease. But as the manifest sample is for bosh lite and we don't need have any HA solution in bosh lite (still single vm/container), didn't include in manifest.

**Comment by sauravmndl** [comment]
Closing this PR. Raised PR in routing-release: https://github.com/cloudfoundry/routing-release/pull/111

**Pull request closed without merging**